src/app/datagrid/selection/selection.html

```diff
@@ -79,7 +83,11 @@ <h3>Client side, trackBy index</h3>
     <ng-container *clrDgHideableColumn> Favorite color </ng-container>
   </clr-dg-column>
 
-  <clr-dg-row *clrDgItems="let user of clientTrackByIndexUsers; trackBy: trackByIndex" [clrDgItem]="user">
+  <clr-dg-row
+    *clrDgItems="let user of clientTrackByIndexUsers; trackBy: trackByIndex"
+    [clrDgItem]="user"
+    [clrDgRowAriaLabel]="'Select row for ' + user.name"
```

Comment:
This updates the demos on the angular docs application to create unique user names.